### PR TITLE
docs: fix wording in docs

### PR DESCRIPTION
**Pull request by rgkirch**
Improve wording

### CODE REVIEW

File: site/docs/mark/tick.md [hunk 1]
```diff
@@ -38,15 +38,15 @@ A tick mark definition can contain any [standard mark properties](mark.html#mark
...
+For example, the following dot plot uses tick marks to show the distribution of rainfall over time.
 
 <span class="vl-example" data-name="tick_dot"></span>
 
 ### Strip Plot
 
 <!-- TODO: better explain this -->
 
-The following strip-plot use `tick` mark to represent its data.
+The following strip-plot use `tick` mark to show the distribution of each car’s Horsepower.
```
Comment (domoritz) [inline site/docs/mark/tick.md:49]:
```suggestion
The following strip-plot use `tick` mark to show the distribution of each car's Horsepower.
```

**Comment by rgkirch** [comment]
image

**Comment by domoritz** [comment]
The second one shows the distribution broken down by number of cylinders. I think the proposed wording is confusing since the chart doesn't show the distribution per car.

File: site/docs/mark/tick.md [hunk 2]
```diff
@@ -38,15 +38,15 @@ A tick mark definition can contain any [standard mark properties](mark.html#mark
...
+For example, the following dot plot uses tick marks to show the distribution of rainfall over time.
 
 <span class="vl-example" data-name="tick_dot"></span>
 
 ### Strip Plot
 
 <!-- TODO: better explain this -->
 
-The following strip-plot use `tick` mark to represent its data.
+The following strip-plot use `tick` mark to show the distribution of each car's Horsepower.
```
Comment (domoritz) [inline site/docs/mark/tick.md:49]:
```suggestion
The following strip-plot use `tick` mark to show the distribution of horsepower.
```

**Pull request merged**